pkgs/tools/inputmethods/uim/default.nix

```diff
@@ -53,16 +53,20 @@ assert withNetworking -> curl != null && openssl != null;
 assert withFFI -> libffi != null;
 assert withMisc -> libeb != null;
 
+let
+  stdenv = if args.stdenv.cc.isGNU then args.gcc13Stdenv else args.stdenv;
+in
+
 stdenv.mkDerivation rec {
-  version = "1.8.8";
+  version = "1.8.9";
   pname = "uim";
 
   src = fetchFromGitHub {
     owner = "uim";
     repo = "uim";
-    rev = "2c0958c9c505a87e70e344c2192e2e5123c71ea5";
```

Comment:
rev -> tag

Reply:
> rev -> tag

Is this new ever since https://github.com/NixOS/nixpkgs/pull/368177 ? How come that PR didn't change `uim`?

Reply:
Only `refs/tags/...` were migrated